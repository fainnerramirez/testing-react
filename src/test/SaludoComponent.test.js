import React from "react";
import "@testing-library/jest-dom";
//import { render, screen } from "@testing-library/react";
//import SaludoComponent from "../components/Saludo.component";

import { shallow } from "enzyme";
import SaludoComponent from "../components/Saludo.component";

describe("pruebas del archivo Saludo.component.js", () => {
  /*test('debe de mostrar el mensaje "Hola que tal!"', () => {
    const saludo = "Hola que tal!";

    render(<SaludoComponent saludo="Hola que tal!" />);

    expect(screen.getByText(saludo)).toBeInTheDocument();
  });*/

  test("debe de mostrar el componente <SaludoComponent /> correctamente", () => {
    const saludo = "Hola que tal!";

    const view = shallow(<SaludoComponent saludo={saludo} />);

    expect(view).toMatchSnapshot();
  });
});
