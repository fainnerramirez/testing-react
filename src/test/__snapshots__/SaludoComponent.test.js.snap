// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas del archivo Saludo.component.js debe de mostrar el componente <SaludoComponent /> correctamente 1`] = `
<div>
  <h1>
    Hola que tal!
  </h1>
  <h3>
    Esto es un subtitulo
  </h3>
</div>
`;
